docs/_data/paths/limits_realtime-snapshot.yaml

```diff
@@ -20,7 +20,7 @@ get:
     '200':
       description: The System Operating Limits snapshot is returned.
       content:
-        application/vnd.trolie.realtime-limits-snapshot.v1+json:
+        application/vnd.trolie.realtime-limits-slim-snapshot.v1+json:
```

Comment:
With Forecast, we distinguished between "slim" and "detailed" by having a `application/vnd.trolie.forecast-limits-detailed-snapshot.v1+json` snapshot to complement the "slim" `application/vnd.trolie.forecast-limits-snapshot.v1+json`. We assumed that the most common (interop) use case would be just getting the limits forecast without needing the details. Anyway, we should be consistent with Forecast here, I think.

Reply:
We have that; this update was simply because I noticed I missed "slim" in the name of the slim media type.  The detailed one is there though.

Reply:
I was pointing out that there is no application/vnd.trolie.forecast-limits-**slim**-snapshot.v1+json

Put another way, we don't use the term *slim* in the slim media type.

Reply:
Oh.  I don't know why I had it in my head that the name "slim" was till applied to the forecast media type.  My bad.